starter-code/pizza.js

```diff
@@ -1 +1,209 @@
 // Write your Pizza Builder JavaScript in this file.
+
+
+
+//when document is loaded, user has to see suce and gluten-free button disabled. Pizza foto has to be without white sauce and gluten-free pizza base.
+
+$(document).ready(function () {
+
+  //whit this code i do not have to edit HTML file and manualy delete class that HTML tag has.
+  //after this two if statements I am sure that buttons want be active.
+  if ($(".btn-sauce").hasClass("active")) {
+    $(".btn-sauce").removeClass("active");
+  }
+
```

Comment:
`$(".btn-sauce").toggleClass("active")` will remove the class if it exists and add it if it does not exist.  Perhaps it might be useful here to be more explicit about why you want to deactivate these buttons.